README.md

```diff
@@ -520,6 +520,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Extirpater**](https://gitlab.com/divested-mobile/extirpater) <sup>**[[F-Droid](https://www.f-droid.org/app/us.spotco.extirpater)]**</sup>
 * [**PCAPdroid**](https://github.com/emanuele-f/PCAPdroid) <sup>**[[F-Droid](https://f-droid.org/app/com.emanuelef.remote_capture)]**</sup>
 * [**UpgradeAll**](https://github.com/DUpdateSystem/UpgradeAll) <sup>**[[F-Droid](https://f-droid.org/app/net.xzos.upgradeall)]**</sup>
+* [**LibChecker**](https://github.com/zhaobozhen/LibChecker)
```

Comment:
The app contains App Center analytics (https://github.com/zhaobozhen/LibChecker/issues/165, thanks @ shuvashish76 for pointing this out).

From a quick search through the code it seems to be Opt-out, without any consensus. Does anyone have more infos here?

Reply:
The privacy policy is also here: https://github.com/zhaobozhen/LibChecker-Docs/blob/master/docs/guide/PRIVACY.md. This app uses Microsoft AppCenter, which is described in the policy.

The policy document appears to be general, like many other Chinese software policy document, and it seems to be in mainly for compliance with the app store regulations, as pointed out by @noarchwastaken in https://github.com/zhaobozhen/LibChecker/issues/261. And the maintainer, @zhaobozhen, said he will improve the representations.

The project is totally free and opensource, only AppCenter included and enabled by default.

Reply:
I can't read the policy and issue unfortunately. 

The App Center libs are MIT licensed ([appcenter-crashes](https://mvnrepository.com/artifact/com.microsoft.appcenter/appcenter-crashes/1.10.0), [appcenter-analytics](https://mvnrepository.com/artifact/com.microsoft.appcenter/appcenter-analytics/2.3.0), [Repo (both)](https://github.com/microsoft/appcenter-sdk-android)) Even though opt-out it's still implemented by open source and would comply with [F-Droids Tracking antifeature](https://f-droid.org/docs/Anti-Features/#Tracking).

So while it has some smell, I'd say it meets the criteria. Any opinions?

Reply:
Updated, thanks!